### PR TITLE
🎪 Fix showing all videos in category

### PR DESCRIPTION
**Pull request by WRadoslaw**
#4010 
#4019

### CODE REVIEW

File: packages/atlas/src/config/contentFilter.ts [hunk 1]
```diff
@@ -3,19 +3,11 @@ import { atlasConfig } from '@/config/config'
 
 import { allUniqueVideoCategories } from './categories'
 
-type ContentFilter = string[]
-
-export const filteredVideoIds: ContentFilter = atlasConfig.content.blockedVideoIds
-export const filteredChannelIds: ContentFilter = atlasConfig.content.blockedChannelIds
-
 export const createChannelWhereObjectWithFilters = (
```
Comment (drillprop) [inline packages/atlas/src/config/contentFilter.ts]: I'd remove this function as well, it's not doing anything actually.

File: packages/atlas/src/config/contentFilter.ts [hunk 2]
```diff
@@ -25,14 +17,11 @@ export const createVideoWhereObjectWithFilters = (
...
 
-    // filter by video ids
-    ...(filteredVideoIds.length ? { id_not_in: filteredVideoIds } : {}),
-
     // filter by channel ids
     channel: createChannelWhereObjectWithFilters(videoWhereInput?.channel),
 
     // filter by category
-    ...(!atlasConfig.content.showAllContent
+    ...(atlasConfig.content.showAllContent
```
Comment (drillprop) [inline packages/atlas/src/config/contentFilter.ts]: I would also remove `showAllContent` from the config. This option is also configurable in Orion. So I don't think we should keep it

Comment (drillprop) [reply packages/atlas/src/config/contentFilter.ts]: Also, let's remove `createVideoWhereObjectWithFilters` as well.

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]
Thanks for cleaning this up, looks good to me!

**Review by attemka — APPROVED** [review]

**Pull request merged**